src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQueryOutputFormat.java

```diff
@@ -589,9 +589,12 @@ private void handleInsertOperation(TableReference tableRef, String writeDisposit
       triggerBigqueryJob(projectId, jobId, dataset, config);
     }
 
-    private void handleUpdateUpsertOperation(TableReference tableRef, @Nullable String cmekKey, JobId jobId,
+    private void handleUpdateUpsertOperation(TableReference tableRef,
```

Comment:
Need to add the param in the calling place as well.

Reply:
oof

Reply:
Added now.